src/main/java/io/codelex/classesandobjects/practice/BankAccount.java

```diff
@@ -0,0 +1,35 @@
+package io.codelex.classesandobjects.practice;
+
+// Each BankAccount object represents one user's account
+// information including name and balance of money.
+
+public class BankAccount {
```

Comment:
`If the balance is negative, put the - sign before the dollar sign. Also, always display the cents as a two-digit number. For example, if the same object had a balance of -17.5,`